src/app/organization/loan-provisioning-criteria/loan-provisioning-criteria.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+  <button mat-raised-button color="primary">
+    <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+    Create Provisioning Criteria
+  </button>
+</div>
+
+<div class="container">
+
+  <div fxLayout="row" fxLayoutGap="20px">
+    <mat-form-field fxFlex>
+      <mat-label>Filter</mat-label>
+      <input matInput (keyup)="applyFilter($event.target.value)">
+    </mat-form-field>
+  </div>
+
+  <div class="mat-elevation-z8">
+
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <ng-container matColumnDef="name">
```

Comment:
Sorting for this column in not working. Identify the reason and correct the same.

Reply:
![pc](https://user-images.githubusercontent.com/28529491/70193290-95403380-16f6-11ea-9aae-bb56e7847632.gif)
Commited and Force pushed, had also changed my local name.

Reply:
For this time, my local name will remain because I initially authored the commit. It should appear next time as my github username.